<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCredWindow</class>
 <widget class="QDialog" name="EditCredWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="edit_save_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit_value">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="name_edit_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_edit_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_edit_value">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
  </widget>
  <widget class="QLabel" name="phone_edit_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер телефона</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_edit_value">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
